src/js/jsx/sections/layers/LayersGroup.jsx

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2014 Adobe Systems Incorporated. All rights reserved.
```

Comment:
The rule is: when you create a new file, use the current year in the copyright header. So, this should be (c) 2016. Also, I think it would be more consistent to name this file `LayerGroup`.